jung-samples/src/main/java/edu/uci/ics/jung/samples/ShowLayouts.java

```diff
@@ -98,7 +98,7 @@ private LayoutChooser(JComboBox<?> jcb, VisualizationViewer<Integer, Number> vv)
     public void actionPerformed(ActionEvent arg0) {
       Layouts layoutType = (Layouts) jcb.getSelectedItem();
       LayoutAlgorithm layoutAlgorithm = createLayout(layoutType);
-      LayoutAlgorithmTransition.animate(vv, layoutAlgorithm);
+      LayoutAlgorithmTransition.apply(vv, layoutAlgorithm);
     }
```

Comment:
I was getting better effects by leaving out the animation step. I  will probably swap between 'animate' and 'apply' from time to time. Animate is more effective between 2 static layouts.